app/assets/javascripts/admin/addon/controllers/admin-revamp.js

```diff
@@ -0,0 +1,31 @@
+import Controller from "@ember/controller";
+import { inject as service } from "@ember/service";
+import { dasherize } from "@ember/string";
+import discourseComputed from "discourse-common/utils/decorators";
+
+export default class AdminRevampController extends Controller {
+  @service router;
+
+  @discourseComputed("router._router.currentPath")
```

Comment:
Not sure if I need all this but just copied it from the overarching admin route for now